src/components/IngredientFeed.vue

```diff
@@ -33,3 +127,79 @@ export default {
   },
 };
 </script>
+
+<style scoped>
+.container {
+  padding: 0 !important;
+  margin: 0 !important;
+  background-color: #f9f9f9;
+  max-width: 100%;
+}
+
+.btn {
+  background-color: white;
+  border-radius: 100px;
+  box-shadow: 0px 3px 3px #bdbdbd;
+}
+
+.tool-bar {
+  margin: 0 !important;
+  flex-wrap: wrap;
+  height: fit-content;
+  display: flex;
+  justify-content: space-between;
+}
+
+.form-outline {
+  padding-right: 20px;
+}
+.form-control {
+  border: none;
+  border-radius: 100px !important;
+  box-shadow: 0px 3px 3px #bdbdbd;
+  display: inline-block;
+}
+
+.tool-bar-container {
+  display: flex;
+  padding: 20px;
+}
+
+.btn-floating {
+  border-radius: 50%;
+  height: fit-content;
+  align-self: center;
+}
+
+.type-of-food {
+  width: 100px;
+  height: 100px;
```

Comment:
Here and in food-item, I think that the size of the items should be proportional to the width of the window. If you go on netflix and resize the window, you'll see that the height and width of the items grow as you make the window wider. Something like that would look really nice IMO if we can figure out how to do that.